src/07-even-more-loops.js

```diff
@@ -96,7 +140,13 @@ function evenArray(array) {}
  *  findSmallest([ 0, 11, -2, 5 ]);
  *  //> -2
  */
-function findSmallest(array) {}
+function findSmallest(array) {
+  let newArr = [];
+  for(let i = 0; i < array.length; i++){
+    newArr.push(array[i]);
+  }
+  return Math.min(...newArr)
```

Comment:
Be careful pulling code from online - if I were to ask you what. ` ...newArr ` returns could you walk me through what that code is doing?